docs/tutorials/services/source-ip.md

```diff
@@ -106,12 +146,21 @@ client_address=10.244.3.8
 command=GET
 ...
 ```
+<!--
 If the client pod and server pod are in the same node, the client_address is the client pod's IP address. However, if the client pod and server pod are in different nodes, the client_address is the client pod's node flannel IP address.
+-->
+如果客户端 pod 和 服务端 pod 在相同的节点上，client\_address 就是客户端 pod 的 IP 地址。但是，如果它们在不同的节点上， client\_address 将会是客户端 pod 的节点的 flannel IP 地址。
```

Comment:
The '\\' is used to escape charactor '\_' in the word 'client_address', otherwise some editor will render contents between both '\_' in these two 'client_address' into italic style mistakenly. As I find it's pretty safe to use it without escaping '\_' here in github, I'll just delete it ^_^.